src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1102,6 +1102,7 @@ RebalanceTableShards(RebalanceOptions *options, Oid shardReplicationModeOid)
 
 	if (list_length(placementUpdateList) == 0)
 	{
+		ereport(NOTICE, (errmsg("There are no rebalance moves for your current cluster.")));
```

Comment:
I don't think this is particularly useful, but it could be a good opportunity to educate the user a little, e.g.
```
NOTICE: shards are already balanced across the nodes using the by_shard_count strategy
HINT: To rebalance shards by disk size, add the following argument: rebalance_strategy := 'by_disk_size'
```

(and do the opposite on the current rebalance strategy, and suggest by_disk_size for other strategies)